### PR TITLE
LL-6449 / NFT Display

### PR DESCRIPTION
**Pull request by klambert-ledger**
All the display features of the NFTs.

### Type

Feature 

### Context

[LL-7132](https://ledgerhq.atlassian.net/browse/LL-7132)
[LL-7136](https://ledgerhq.atlassian.net/browse/LL-7136)
[LL-7138](https://ledgerhq.atlassian.net/browse/LL-7138)
[LL-7140](https://ledgerhq.atlassian.net/browse/LL-7140)
[LL-7145](https://ledgerhq.atlassian.net/browse/LL-7145)
[LL-7147](https://ledgerhq.atlassian.net/browse/LL-7147)
[LL-7149](https://ledgerhq.atlassian.net/browse/LL-7149)

### Parts of the app affected / Test plan

Single Account Page

### CODE REVIEW

File: src/screens/OperationDetails/Content.js [hunk 1]
```diff
@@ -148,32 +225,7 @@ export default function Content({
...
-          <DoubleCounterValue
-            showCode
-            alwaysShowSign
-            currency={currency}
-            value={amount}
-            date={operation.date}
-            subMagnitude={1}
-          />
-        )}
+        {operationTitleRender()}
```
Comment (gre) [inline src/screens/OperationDetails/Content.js:228]: a bit anti-pattern / overcomplex part

I would see this to be more a component than a function.

File: ios/Podfile [hunk 2]
```diff
@@ -16,7 +16,6 @@ target 'ledgerlivemobile' do
     :hermes_enabled => false
   )
 
-  pod 'djinni_objc', :path => '../node_modules/@ledgerhq/react-native-ledger-core'
```
Comment (gre) [inline ios/Podfile:19]: i'm not sure why this was committed here.

File: src/icons/ImageNotFound.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
+import Svg, { Path } from "react-native-svg";
+
+type Props = {
+  height?: number,
+  width?: number,
+  color?: string,
+};
+
+const ImageNotFound = ({ height, width, color }: Props) => (
```
Comment (gre) [inline src/icons/ImageNotFound.js:10]: please make this a pure component.

File: src/icons/OpenSea.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
+import Svg, { G, Rect, Path, Defs, ClipPath } from "react-native-svg";
+
+type Props = {
+  size?: number,
+};
+
+const OpenSeaIcon = ({ size }: Props) => (
```
Comment (gre) [inline src/icons/OpenSea.js:8]: same

File: src/icons/Globe.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import * as React from "react";
+import Svg, { Path } from "react-native-svg";
+
+type Props = {
+  size?: number,
+  color?: string,
+};
+
+const GlobeIcon = ({ size, color }: Props) => (
```
Comment (gre) [inline src/icons/Globe.js:9]: same

File: src/icons/Rarible.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import Svg, { G, Path, Defs, ClipPath } from "react-native-svg";
+
+type Props = {
+  size?: number,
+};
+
+const RaribleIcon = ({ size }: Props) => (
```
Comment (gre) [inline src/icons/Rarible.js:8]: same

File: src/screens/Nft/NftCollection/index.js [hunk 7]
```diff
@@ -0,0 +1,235 @@
...
+          nativeEvent: {
+            contentOffset: { y: scrollY },
+          },
+        },
+      ],
+      { useNativeDriver: true },
+    );
+
+  // nfts' list related -----
+  const [nftCount, setNftCount] = useState(12);
```
Comment (gre) [inline src/screens/Nft/NftCollection/index.js:69]: instead of an arbitrary `12` please make this a constant in that file to "document it".

**Review by gre — DISMISSED** [review]
we have perf challenge to tackle later (the fact that very single eth account sync re-render the complete nft list will be problematic).
but i think we can nearly ship this if it pass QA as a first milestone.

File: ios/Podfile (hunk 2 above)
Comment (klambert-ledger) [reply ios/Podfile:19]: 🤷‍♂️ I had to do some pod install after the last rebase and I guess that was added by one of the rebase ?

I'll just remove it if you think it shouldn't be here

Comment (klambert-ledger) [reply ios/Podfile]: Done 👍

File: src/screens/OperationDetails/Content.js (hunk 1 above)
Comment (klambert-ledger) [reply src/screens/OperationDetails/Content.js]: Added a React.memo component for that 👌

File: src/icons/ImageNotFound.js (hunk 3 above)
Comment (klambert-ledger) [reply src/icons/ImageNotFound.js:10]: Added a React.memo on the export 👌

File: src/icons/OpenSea.js (hunk 4 above)
Comment (klambert-ledger) [reply src/icons/OpenSea.js:8]: Added a React.memo on the export 👌

File: src/icons/Rarible.js (hunk 6 above)
Comment (klambert-ledger) [reply src/icons/Rarible.js:8]: Added a React.memo on the export 👌

File: src/icons/Globe.js (hunk 5 above)
Comment (klambert-ledger) [reply src/icons/Globe.js:9]: Added a React.memo on the export 👌

File: src/screens/Nft/NftCollection/index.js (hunk 7 above)
Comment (klambert-ledger) [reply src/screens/Nft/NftCollection/index.js]: Added constants on NFTCollection and NFTGallery screens 👌

**Review by gre — APPROVED** [review]

**Pull request merged**